app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -27,6 +27,8 @@ def perform(form526_submission_id)
 
       begin
         with_tracking(self.class.name, form526_submission.saved_claim_id, form526_submission_id) do
+          return if pending_eps?(form526_submission.auth_headers)
```

Comment:
I believe Yang found a possible bug: https://github.com/department-of-veterans-affairs/vets-api/pull/9344/files#r826368786

Reply:
Todo: change to `next`

Reply:
My understanding is that we should always be using `next` instead of `return` for these blocks to function correctly. Going to update the `return` statement a few lines down too!

Reply:
We should prob add a code comment to prevent someone (like me) from changing it back to `return`.

Reply:
Yup that's what I meant in my previous comment